### PR TITLE
[ShadCN] refactor(pages/what-is-ethereum): migrate to ShadCN

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description

Migrates the "What is Ethereum" page to remove direct Chakra imports and use ShadCN components and Tailwind.

### CODE REVIEW

**Comment by TylerAPfledderer** [comment]
I would say that the [Learn Page PR](https://github.com/ethereum/ethereum-org-website/pull/13569) is a blocker here; it is establishing the import of the new buttons and links, which means movement of various files.

Once that is done and merged, then update this PR and those imports for the "What is Ethereum" page.

**Comment by Wachira3712** [comment]
Wachira3712

**Comment by TylerAPfledderer** [comment]
@pettinarip should be good now

File: src/pages/what-is-ethereum.tsx [hunk 1]
```diff
@@ -72,117 +61,103 @@ import stats from "@/public/images/upgrades/newrings.png"
...
-    as="h1"
-    fontSize="sm"
-    lineHeight={1.4}
-    letterSpacing="wider"
-    fontWeight="500"
-    mb={4}
-    textTransform="uppercase"
-    color="textTableOfContents"
+  <h1
+    className="text-sm font-medium uppercase leading-xs tracking-wider text-[#7f7f7f] dark:text-[hsla(0,0%,69.8%,.8)]"
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:69]: Validated with Nuno, lets use `body` color for this
```suggestion
    className="text-sm font-medium uppercase leading-xs tracking-wider"
```

File: src/pages/what-is-ethereum.tsx [hunk 2]
```diff
@@ -72,117 +61,103 @@ import stats from "@/public/images/upgrades/newrings.png"
...
     {...props}
   />
 )
 
 const Subtitle = (props: ChildOnlyProp) => (
-  <Text fontSize="xl" lineHeight={1.4} color="text200" {...props} />
+  <p
+    className="text-xl leading-xs text-[#666] dark:text-[#b2b2b2]"
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:76]: same here
```suggestion
    className="text-xl leading-xs"
```

File: src/pages/what-is-ethereum.tsx [hunk 3]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+                        <p>{t("page-what-is-ethereum-summary-desc-1")}</p>
+                        <p>{t("page-what-is-ethereum-summary-desc-2")}</p>
+                        <p>{t("page-what-is-ethereum-summary-desc-3")}</p>
+                      </Stack>
+                    </Stack>
+                  </Summary>
+                </Width60>
+                <Width40 />
+              </TwoColumnContent>
+              <div>
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:352]: This was a missing space we needed to add
```suggestion
              <Stack className="gap-8">
```

File: src/pages/what-is-ethereum.tsx [hunk 4]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+            </Stack>
+          </Section>
+
+          <Section>
+            <TwoColumnContent>
+              <Width60>
+                <H2>
+                  {t("page-what-is-ethereum-why-would-i-use-ethereum-title")}
+                </H2>
+                <Stack className="gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:396]:
```suggestion
                <Stack className="gap-6">
```

File: src/pages/what-is-ethereum.tsx [hunk 5]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+                      trackCustomEvent({
+                        eventCategory: `What is Ethereum - Slider`,
+                        eventAction: `Clicked`,
+                        eventName: slides[index].eventName,
+                      })
+                    }}
+                  >
+                    <EmblaSlide>
+                      <H3>{t("page-what-is-ethereum-slide-1-title")}</H3>
+                      <Stack className="mb-[1.45rem] gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:411]:
```suggestion
                      <Stack className="mb-4 gap-6">
```

File: src/pages/what-is-ethereum.tsx [hunk 6]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+                        <p>
+                          <Translation id="page-what-is-ethereum:page-what-is-ethereum-slide-1-desc-1" />
+                        </p>
+                        <p>{t("page-what-is-ethereum-slide-1-desc-2")}</p>
+                      </Stack>
+                    </EmblaSlide>
+                    <EmblaSlide>
+                      <H3>{t("page-what-is-ethereum-slide-2-title")}</H3>
+                      <Stack className="mb-[1.45rem] gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:420]:
```suggestion
                      <Stack className="mb-4 gap-6">
```

File: src/pages/what-is-ethereum.tsx [hunk 7]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+                        <p>{t("page-what-is-ethereum-slide-2-desc-1")}</p>
+                        <p>
+                          <Translation id="page-what-is-ethereum:page-what-is-ethereum-slide-2-desc-2" />
+                        </p>
+                      </Stack>
+                    </EmblaSlide>
+                    <EmblaSlide>
+                      <H3>{t("page-what-is-ethereum-slide-3-title")}</H3>
+                      <Stack className="mb-[1.45rem] gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:429]:
```suggestion
                      <Stack className="mb-4 gap-6">
```

File: src/pages/what-is-ethereum.tsx [hunk 8]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+                        <p>
+                          <Translation id="page-what-is-ethereum:page-what-is-ethereum-slide-3-desc-1" />
+                        </p>
+                      </Stack>
+                    </EmblaSlide>
+                    <EmblaSlide>
+                      <H3>{t("page-what-is-ethereum-slide-4-title")}</H3>
+                      <Stack className="mb-[1.45rem] gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:437]:
```suggestion
                      <Stack className="mb-4 gap-6">
```

File: src/pages/what-is-ethereum.tsx [hunk 9]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+          </Section>
+
+          <Section className="bg-[#e8e8ff] dark:bg-[#212131]">
+            <TwoColumnContent>
+              <Width40>
+                <Image400 src={whoRunsEthereum} />
+              </Width40>
+              <Width60>
+                <H2>{t("page-what-is-ethereum-who-runs-ethereum-title")}</H2>
+                <Stack className="gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:618]:
```suggestion
                <Stack className="gap-6">
```

File: src/pages/what-is-ethereum.tsx [hunk 10]
```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
...
+          </Section>
+
+          <Section>
+            <TwoColumnContent className="lg:flex-row-reverse">
+              <Width40>
+                <Image400 src={whatAreSmartContracts} />
+              </Width40>
+              <Width60>
+                <H2>{t("page-what-is-ethereum-smart-contract-title")}</H2>
+                <Stack className="gap-[1.45rem]">
```
Comment (pettinarip) [inline src/pages/what-is-ethereum.tsx:642]:
```suggestion
                <Stack className="gap-6">
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
@TylerAPfledderer looks good overall. Left a few comments to try to standarize some colors and spacing values. Didn't add a comment in all of the occurrences but you will get the point. Lets see if we can use the tw scale as much as we can.

**Review by pettinarip — APPROVED** [review]
Nice job @TylerAPfledderer.

Will validate a few things with Nuno and then merge it.

**Pull request merged**